templates/index.html

```diff
@@ -51,32 +54,34 @@
 
         }
 
-        function got_schedds(data) {
+        function got_schedds(data, attribs) {
           $('#info_table_inner').detach();
           info_table = $("<table></table>");
           info_table.addClass("table");
           info_table.addClass("tablesorter");
           info_table.attr("id", "info_table_inner");
-          $.getJSON("/json/tableattribs", function(attribs) {
-            info_table.append("<thead>");
-            info_table.append("<th>CE</th>");
-            attribs.forEach(function(it) {
-                info_table.append("<th>" + it.label + "</th>");
-            });
-            info_table.append("</thead>");
 
-            info_table.append("<tbody>");
-            for (name in data) {
-                key = data[name]
-                info_table.append("<tr>");
-                info_table.append("<td>" + name + "</td>");
-                attribs.forEach(function(it) {
-                    info_table.append("<td>" + key[it.attrib] + "</td>");
-                });
-                info_table.append("</tr>");
-            }
-            info_table.append("</tbody>");
+          thead = $('<thead>');
+          row = $('<tr/>');
```

Comment:
Uh does this work? `<tr>` is certainly not an empty tag.

Reply:
My understanding is that append() puts data inside an element, rather than after.

The intent was to create a row element, then use append() to fill it with data cells.